### ISSUE TITLE
Price updates seem to be for american stock exchange opening times?

### ISSUE THREAD

**Issue by Satlinker** [premnirmal/StockTicker#250]
Great app. It seems to me, that the prices are only updated correct when exchanges in USA are open? You can do a request for other times, but prices differ from e.g. european exchanges?

**Comment by flurbius** [comment]
Yes I find that having just one set of times is too restrictive - I would like to see each widget having its own active times, and would be prepared to do it.
